pkg/konfluxgen/konfluxgen.go

```diff
@@ -181,7 +181,9 @@ func Generate(cfg Config) error {
 				Nudges:                        append(cfg.Nudges, cfg.NudgesFunc(c.ReleaseBuildConfiguration, ib)...),
 				Pipeline:                      pipeline,
 				AdditionalTektonCELExpression: cfg.AdditionalTektonCELExpressionFunc(c.ReleaseBuildConfiguration, ib),
+				Tags:                          []string{"latest"},
```

Comment:
I probably miss something, but just to be sure here: as we use the same pipeline for on-push and on-pr, wouldn't we tag images from PRs also with `latest`?

Reply:
no, the parameter is only added for `push` pipeline

Reply:
```
    {{{- if eq .Event "push" }}}
    - name: additional-tags
      value: [{{{ range $tag := .Tags }}} {{{ $tag }}}, {{{ end }}}]
    {{{- end }}}
```

Reply:
we're injecting parameters for the pipeline run and we have one pipeline run per event (push and PR separate)

Reply:
Ah nice :+1: